app/views/newflow/login_signup/welcome.html.erb

```diff
@@ -1,3 +1,7 @@
+<%
+conditional_educator_signup_path = Settings::Db.store.educator_feature_flag ? educator_signup_path : signup_path(bpff: 9)
```

Comment:
can we name this something better than bpff? :)

Reply:
not a big deal here...but would be great to put conditional logic in a helper so that it can be easily tested.

Reply:
Sure! Feel free to create a PR or make the change in one of your PRs.